scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -198,19 +214,19 @@ public boolean doCheckAndSend(boolean withAttachment)
 
     }
 
-    private void sendEmail(boolean withAttachment, boolean withErrors)
+    private void sendEmail(boolean withAttachment, boolean withErrors, boolean aborted)
```

Comment:
Maybe providing a dedicated Java structure accounting for of a Job status can be more sustainable than providing a set of boolean parameter to a function: If we are to extend later the mail notification to other type of Job (or even other attributes of a job !), it will require less refactoring (1 structure edition vs. N method signatures to adapt).

Reply:
done